scripts/microbe_stage/organelle.as

```diff
@@ -725,13 +719,11 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
 
     ObjectID microbeEntity = NULL_OBJECT;
     ObjectID organelleEntity = NULL_OBJECT;
+    Species@ species;
```

Comment:
Maybe this could be a handle to a const (to not allow the organelle to accidentally change the species properties).